src/main/java/org/folio/consortia/domain/entity/base/AuditableEntity.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.consortia.domain.entity.base;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import java.sql.Timestamp;
```

Comment:
java.sql.Timestamp is legacy class, preferable to use java.time.LocalDateTime

Reply:
fixed